src/Config.ts

```diff
@@ -4,9 +4,9 @@
  * 
  *!*/
 
-import * as Path from 'path';
 import * as TSD from './TypeScriptDeclarations';
 import * as Utils from './Utilities';
+import { sep, posix } from 'path';
```

Comment:
Would be great, if you stick to alphabetical order.

Reply:
Sure thing 👍 Please see latest changes @bre1470